examples/decimal-type/decimal_type.description

```diff
@@ -1,10 +1,6 @@
-// It is the third numeric type. Works like `int` and `float`. No implicit conversion.
-// Represents `decimal` fractions exactly. Avoids surprises that you get with `float`. Preserves precision.
-// Separate basic type; counts as `anydata`. Literal uses the suffix `d`(`f` suffix is for `float`).
-// Floating point, not infinite precision.
-// <ul>
-// <li>34 decimal digits</li>
-// <li>22 digits are enough for US national debt in `¢`</li>
-// <li>27 digits are enough for an age of universe in `ns`</li>
-// <li>No `infinity`, `NaN` or negative zero</li>
-// </ul>
+// `decimal` type is the third numeric type. It works like `int` and `float` types.
+// It is a separate basic type and counts as `anydata` type.
+// No implicit conversions between `decimal` and other numeric types is allowed.
+// It can represent `decimal` fractions exactly and it preserves precision.
+// `decimal` type does not include `infinity`, `NaN` or negative zero.
+// It supports floating point precision to 34 decimal digits.
```

Comment:
```suggestion
// `decimal` type is the third numeric type. It works like `int` and `float` types.
// It is a separate basic type and counts as `anydata` type.
// Implicit conversions between `decimal` and other numeric types is not allowed.
// It can represent `decimal` fractions exactly and it preserves precision.
// `decimal` type does not include `infinity`, `NaN` or negative zero.
// It supports floating-point precision to 34 decimal digits.
```